internal/config/hook.go

```diff
@@ -23,4 +23,12 @@ type RuntimeHookConfig struct {
 	Path string `toml:"path"`
 	// SkipModeDetection disables the mode check for the runtime hook.
 	SkipModeDetection bool `toml:"skip-mode-detection"`
+	// DisabledHooks specifies the hooks that are disabled for the NVIDIA
+	// Container Runtime hook.
+	DisabledHooks []string `toml:"disabled,omitempty"`
+	// EnabledHooks specifies the hooks that are enabled for the NVIDIA
+	// Container Runtime hook.
+	// Specify a list of explicitly enabled hooks. If a hook is specified as
+	// both enabled and disabled, it will be enabled.
+	EnabledHooks []string `toml:"enabled,omitempty"`
```

Comment:
I don't think we should add these config options, and IF we decide to add them we should not add them in the `nvidia-container-runtime-hook` section. This section is specific for the `nvidia-container-runtime-hook` that is used to call out to the `nvidia-container-cli`.

Reply:
removed for now, maybe a discuss-todo item